backend/database/actionsWithTables/productActions.js

```diff
@@ -0,0 +1,40 @@
+const { Product } = require('../tables/product');
+
+const addNewProduct = async (data) => {
+    try {
+        return await Product.create({
+            name: data.name,
+            price: data.price,
+            unit: data.unit,
+            subcategory_id: data.subcategory_id,
+            producer_id: data.producer_id,
+        });
+    } catch (error) {
+        console.log(error);
+    }
+};
+
+const returnProducts = async () => {
+    const products = await Product.findAll({
+        attributes: ['product_id', 'name', 'price', 'unit', 'subcategory_id', 'producer_id', 'is_deleted'],
+        where: {
+            is_deleted: false,
+        }
```

Comment:
Consider using Sequelize join to fetch related data in one query
```suggestion
        attributes: ['product_id', 'name', 'price', 'unit', 'is_deleted'],
        where: {
            is_deleted: false,
        },
        include: [Subcategory, Producer]
```